themes/vuetober/pages/index.htm

```diff
@@ -10,24 +9,4 @@
     ];
 }
 ==
-
-<!DOCTYPE html>
-<html>
-    <head>
-        
-        <meta charset="utf-8">
-        <meta content="IE=edge" http-equiv="X-UA-Compatible">
-        <meta name="viewport" content="width=device-width,initial-scale=1">
-        <meta name="vuetober" content="{{ vuetober | json_encode }}">
-
-        
-        <title>Vuetober</title>
-    <link href="http://localhost:8080/about.ebfabe1115e3c5bdec33.hot-update.js" rel="prefetch"><link href="http://localhost:8080/js/about.js" rel="prefetch"><link href="http://localhost:8080/js/app.js" rel="preload" as="script"><link href="http://localhost:8080/js/chunk-vendors.js" rel="preload" as="script"></head>
-    <body>
-        <noscript>
-            <strong>We're sorry but Vuetober doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
-        </noscript>
-        <div id="app"></div>
-        
-    <script type="text/javascript" src="http://localhost:8080/js/chunk-vendors.js"></script><script type="text/javascript" src="http://localhost:8080/js/app.js"></script></body>
-</html>
+<!DOCTYPE html><html><head><meta charset=utf-8><meta content="IE=edge" http-equiv=X-UA-Compatible><meta name=viewport content="width=device-width,initial-scale=1"><meta name=vuetober content="{{ vuetober | json_encode }}"><title>Vuetober</title><link href=/themes/vuetober/assets/css/about.abeee0df.css rel=prefetch><link href=/themes/vuetober/assets/js/about.7b3438c9.js rel=prefetch><link href=/themes/vuetober/assets/css/app.9a0dc92f.css rel=preload as=style><link href=/themes/vuetober/assets/js/app.131484b4.js rel=preload as=script><link href=/themes/vuetober/assets/js/chunk-vendors.7e8fd2fe.js rel=preload as=script><link href=/themes/vuetober/assets/css/app.9a0dc92f.css rel=stylesheet></head><body><noscript><strong>We're sorry but Vuetober doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/themes/vuetober/assets/js/chunk-vendors.7e8fd2fe.js></script><script src=/themes/vuetober/assets/js/app.131484b4.js></script></body></html>
```

Comment:
Toto je takto sformatovane naschval?

Reply:
To vzdy spravi. Neviem ci je to eslintom alebo to takto generuje.